hawkbit-repository/hawkbit-repository-jpa/src/main/resources/db/migration/DB2/V1_12_8__change_length_of_created_last_modified_by___DB2.sql

```diff
@@ -0,0 +1,45 @@
+ALTER TABLE sp_action ALTER COLUMN created_by SET DATA TYPE VARCHAR(64);
+ALTER TABLE sp_action ALTER COLUMN last_modified_by SET DATA TYPE VARCHAR(64);
```

Comment:
I think we could alter multiple columns at once using one SQL query:
ALTER TABLE sp_action ALTER COLUMN
   created_by SET DATA TYPE VARCHAR(64),
   last_modified_by SET DATA TYPE VARCHAR(64);

Reply:
Applies to all queries below...